reactnative/RNPlayground/app/navigators/AppNavigator.tsx

```diff
@@ -33,36 +32,33 @@ import { navigationRef, useBackButtonHandler } from "./navigationUtilities"
  *   https://reactnavigation.org/docs/typescript#type-checking-the-navigator
  *   https://reactnavigation.org/docs/typescript/#organizing-types
  */
-export type AppStackParamList = {
-  Welcome: undefined
+export type NavigatorParamList = {
+  Welcome: undefined,
+  About: undefined,
+  Playground: undefined,
   // 🔥 Your screens go here
 }
 
+const Tab = createMaterialBottomTabNavigator<NavigatorParamList>()
+
+function AppTabs() {
+    return (
+        <Tab.Navigator
+          screenOptions={{ headerShown: false }}
+          initialRouteName="Welcome">
+              <Tab.Screen name="Welcome" component={WelcomeScreen} options={{title: "Home"}} />
+              <Tab.Screen name="About" component={AboutScreen} options={{title: "About"}} />
+              <Tab.Screen name="Playground" component={PlaygroundScreen} options={{title: "Playground"}} />
```

Comment:
As you are using the i18 you might want to use it here for these titles

Reply:
With `screenOptions={{ headerShown: false }}`, these titles are not displayed on the UI. However, this title may be readable via accessibility tools, so will check that case and localise if needed.